meta-iot2000-example/recipes-node/node-red-contrib-iiot-opcua/node-red-contrib-iiot-opcua_3.12.1.bb

```diff
@@ -3,7 +3,7 @@ SUMMARY = "The next evolution IoT/IIoT OPC UA toolbox package for Node-RED based
 # WARNING: the following LICENSE and LIC_FILES_CHKSUM values are best guesses - it is
 # your responsibility to verify that the values are complete and correct.
 LICENSE = "Apache-2.0 & ISC & BSD & BSD-3-Clause & MIT"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=fb4562eee6b20e9a23a17f38a9f2527a"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=2419952b1e443ba100cb1c97b931ddab"
```

Comment:
Any change in the licenses as well, given that the checksum changed?

Reply:
They just add a new single year (2019) to the copyright line, the license stays the same.

Reply:
However, the actual license is BSD-3-Clause, should we change the above line to 
```
LICENSE = "BSD-3-Clause"
```
so that it would be more accurate to reflect the license status?

Reply:
Before removing licenses, better to a grep on the code and also the dependencies we package.

Reply:
Do we really need the license of the dependencies (i.e. node_modules)? I checked some community nodes layers, seems just showing the license of the node itself would be satisfied. For example https://github.com/imyller/meta-nodejs-contrib/tree/master/recipes-extended.

Reply:
We are also packaging dependencies, so this recipe should describe it all.

BTW, node-red-contrib-iiot-opcua should be replaced with node-red-contrib-opcua anyway as it's no longer maintained as OSS.